services/common/serv_helper.js

```diff
@@ -5,11 +5,111 @@
     console.log('-*-*- ServHelper: ' + str);
   }
 
+  var ACCESS_CTRL_FILE = 'acl.json';
+
+  function getACL(aUrl) {
+    function aclPathFromURL(aUrl) {
+      if (!aUrl) {
+        debug('Trying to recover acl file from no url');
+        return {};
+      }
+      // Error ctrl?
+      var url = new URL(aUrl);
+      var pathname = url.pathname;
+      return pathname.substring(0, pathname.lastIndexOf('/') + 1);
+    }
+    return new Promise((resolve, reject) => {
+      var xhr = new XMLHttpRequest();
+      xhr.open('GET', aclPathFromURL(aUrl) + ACCESS_CTRL_FILE, true);
+      xhr.responseType = 'json';
+
+      xhr.onerror = function(error) {
+        reject(error);
+      };
+      xhr.onload = function() {
+        if (xhr.response !== null) {
+          var response = xhr.response;
+          for (var developer in response) {
+            var allowedOrgs = response[developer].allowedOrigins;
+            if (!allowedOrgs) {
+              reject(new Error('Wrong acl file. Developer ' + developer +
+                     ' does not have allowedOrigins keys'));
+            }
+            var allowedOrgsRE = [];
+            allowedOrgs.forEach(org => allowedOrgsRE.push(new RegExp(org)));
```

Comment:
for (var i = allowedOrgs.length; i >= 0; i--) allowedOrgs[i] = new RegExp(allowedOrgs[i]);

should be faster and doesn't need a new array.